### PR TITLE
BaSP - MR-112:  Members - home, profile and activities page

### PR DESCRIPTION
**Pull request by JuanIgnacioSobral**
Implemented activities list, homepage, profile page and general CSS. Connected profile with edit profile, but page seems to crash due to change on the shared inputs file.

### CODE REVIEW

**Review by DelfiLT — CHANGES_REQUESTED** [review]
Try to maintain a consistency with the hole app when making the design, use the colors `#47BA8A`  `#373867`, also you should use the same background than the other pages, try to grab all the profile information into a blue box equal to the other ones too, you can copy the properties from the existing
`Background Color: rgba(55,56,103,.8) ` and `Box-shadow: 0 4px 4px rgba(0,0,0,.25);`
And, for the structure, you should adjust the buttons to be in one place, in general the edit profile buttons are on top or in the botom, we should ask the QA team where they want us to put them. And the active state maybe it shouldn't be a button, it could be just a logo or something.

**Review by Milicerro — COMMENTED** [review]
Please, make the profile of the members taking into account what is implemented in the admin profile. 
Slack_xTV8CF0A50

File: src/Components/MembersHome/Activities/activities.module.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  justify-content: space-evenly;
+  width: 95vw;
+  margin-bottom: 75px;
+}
+
+.activity {
+  border-radius: 13px;
+  box-shadow: 0px 14px 7px 1px rgb(0 0 0 / 75%);
+  background-color: #474885;
+  width: 20vw;
```
Comment (DelfiLT) [inline src/Components/MembersHome/Activities/activities.module.css:35]: Please try to replace the width and height with:
```
height: 400px;
width: 15vw;
```

Comment (DelfiLT) [reply src/Components/MembersHome/Activities/activities.module.css:35]: Add the margin bottom because when there are too many cards they stick together

Comment (JuanIgnacioSobral) [reply src/Components/MembersHome/Activities/activities.module.css:35]: Done

**Review by DelfiLT — APPROVED** [review]

**Review by DelfiLT — COMMENTED** [review]
Before you merge, you have to apply the members edit form for the profile

**Review by SantiH97 — CHANGES_REQUESTED** [review]
The activities tab is not working, it just wont load at all

**Comment by DelfiLT** [comment]
> The activities tab is not working, it just wont load at all

It's working, there was no data thats why

**Review by SantiH97 — APPROVED** [review]

File: src/Components/Members/Edit/EditMembers.jsx
Comment (DelfiLT) [inline src/Components/Members/Edit/EditMembers.jsx:1]: You have to add another condition in the if(success) wheter you are in members or in admins, and then change the redirect depending on what entity you are

**Pull request merged**